options/options.c

```diff
@@ -1142,6 +1148,7 @@ static const struct MPOpts mp_default_opts = {
         "fullscreen",
         "ontop",
         "border",
+        "window-affinity",
```

Comment:
Should this really be added as a default watch later option? Seems weird.

Reply:
watch later options are triggered when you change an option while the player is running right? If so then yes, because we can dynamically change the affinity, if not, then i made a mistake there, i was confused with the git stuff

Reply:
They are not. It's for when you use `--save-position-on-quit`.

Reply:
You're right, forgot about that, removed :)